Packs/Base/Scripts/DBotTrainClustering/DBotTrainClustering.yml

```diff
@@ -0,0 +1,153 @@
+args:
+- default: false
+  description: Comma-separated list of incident fields to take into account when training
+    the clustering.
+  isArray: false
+  name: fieldsForClustering
+  required: false
+  secret: false
+- default: false
+  description: Incident field which represent the family name for each cluster created.
+  isArray: true
+  name: fieldForClusterName
+  required: false
+  secret: false
+- default: false
+  defaultValue: 1 month ago
+  description: The start date by which to filter incidents. Date format will be the
+    same as in the incidents query page, for example, "3 days ago", ""2019-01-01T00:00:00
+    +0200").
+  isArray: false
+  name: fromDate
+  predefined:
+  - ''
+  required: false
+  secret: false
+- default: false
+  description: The end date by which to filter incidents. Date format will be the
+    same as in the incidents query page, for example, "3 days ago", ""2019-01-01T00:00:00
+    +0200").
+  isArray: false
+  name: toDate
+  required: false
+  secret: false
+- default: false
+  defaultValue: '3000'
+  description: The maximum number of incidents to query.
+  isArray: false
+  name: limit
+  required: false
+  secret: false
+- default: false
+  description: Argument for the query.
+  isArray: false
+  name: query
+  required: false
+  secret: false
+- default: false
+  defaultValue: '2'
+  description: Minimun number of incident a cluster should contain for it to be retained.
+  isArray: false
+  name: minNumberofIncidentPerCluster
+  required: false
+  secret: false
+- default: false
+  description: Name of the model
+  isArray: false
+  name: modelName
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'False'
+  description: Whether to store the model in the system.
+  isArray: false
+  name: storeModel
+  predefined:
+  - 'True'
+  - 'False'
+  required: false
+  secret: false
+- default: false
+  defaultValue: '0.4'
```

Comment:
Like discussed I think this argument can be confusing as it depends on number of unique fields values, and can cause  unexpected behaviour when number of unique values is high (probably no clusters will achieved for 4 unique values or more, and customer won't know he needs to play with this argument).
 Although it's complicated, would prefer if we calculate this by our own, perhaps something like `1/N_UNIQUE_VALUES` give or take

Reply:
talked already

Reply:
Wrote it after the talk, still think it should take into account number of unique values of fields, or it might be hard to understand when no clusters are returned for 4 or more unique values